README.md

```diff
@@ -32,19 +32,7 @@ const knockClient = new Knock("sk_12345");
 
 ## Usage
 
-### Identifying users
-
-```javascript
-const { Knock } = require("@knocklabs/node");
-const knockClient = new Knock("sk_12345");
-
-await knockClient.users.identify("jhammond", {
-  name: "John Hammond",
-  email: "jhammond@ingen.net",
-});
-```
-
-### Sending notifies (triggering workflows)
+### Sending notifications (triggering workflows)
```

Comment:
Just reworded and moved around some sections to make it more consistent but didn't add anything new here